crates/rolldown_common/src/types/symbol_ref_db.rs

```diff
@@ -60,21 +64,30 @@ impl SymbolRefDbForModule {
     }
   }
 
-  pub fn create_symbol(
-    &mut self,
-    span: Span,
-    name: CompactString,
-    flags: SymbolFlags,
-    scope_id: ScopeId,
-    node_id: NodeId,
-  ) -> SymbolId {
+  // The `facade` means the symbol is actually not exist in the AST.
```

Comment:
`facade` not have `not exists` meaning in English, I prefer `dummy`

Reply:
you could refer here https://github.com/rolldown/rolldown/pull/2447/files#diff-a572f77d3befed0f4403b2a59164bd5179e5daa68ac58d2c74597e5fbc68e563R80

Reply:
> `facade` not have `not exists` meaning in English, I prefer `dummy`

Or say, it did not widely used for meaning `not exists`, that's why I recommend `dummy`, just like `oxc` did